File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.cpp
#define BIT_LEN 32
class Solution {
public:

    int countNumOfOneBit(int n){
        int count = 0;
        for(;n;n=n>>1)  count += n%2;
        return count;
    }
    int minBitFlips(int start, int goal) {       
        return countNumOfOneBit(start ^ goal);
    }
};

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
BIT_LEN = 32
class Solution:
    def count_set_bits(self,n):
        count = 0
        while n:
            count += n%2
            n = n>>1
        return count
    def minBitFlips(self, start: int, goal: int) -> int:
        return self.count_set_bits(start ^ goal)
        